endpoints.json

```diff
@@ -1,4 +1,5 @@
 {
+
   "GET /api": {
     "description": "serves up a json representation of all the available endpoints of the api"
```

Comment:
Make sure to flesh this out as you add more endpoints. 

Include for each endpoint:
 A description of the purpose and functionality of the endpoint.
 Any available queries (can be an empty array if there are none).
 What format the request body needs to take to be accepted (if applicable).
 An example response.